jnigen/test/simple_package_test/java/com/github/dart_lang/jnigen/simple_package/Example.java

```diff
@@ -176,12 +176,27 @@ public Nested(boolean value) {
       this.value = value;
     }
 
+    public void usesAnonymousInnerClass() {
```

Comment:
Any reason to add this method? It seems not to be tested.

Reply:
It uses an anonymous inner class, which, when compiled, creates a file `Example$Nested$1.class`.

The presence of this class in test ensures presence of anonymous classes don't break code gen.